addons/editor/initKeybinds.sqf

```diff
@@ -21,6 +21,24 @@
             [_x] call EFUNC(common,ejectPassengers);
         } forEach SELECTED_OBJECTS;
 
-        true // handled, prevents vanilla eject from activating
+        true // handled, prevents vanilla eject
     };
 }, {}, [DIK_G, [false, true, false]]] call CBA_fnc_addKeybind; // Default: CTRL + G
+
+[ELSTRING(common,Category), QGVAR(deepCopy), [LSTRING(DeepCopy), LSTRING(DeepCopy_Description)], {
+    if (!isNull curatorCamera && {!GETMVAR(RscDisplayCurator_search,false)}) then {
+        GVAR(clipboard) = [SELECTED_OBJECTS] call FUNC(deepCopy);
+        playSound ["RscDisplayCurator_error01", true];
```

Comment:
Why is it playing the error sound? It could play the `FD_Finish_F` for better UX effect.

Reply:
Despite the name, this is the ping sound played when a Zeus message is shown.